mlir/test/Dialect/Tosa/tosa_validation_init.mlir

```diff
@@ -0,0 +1,21 @@
+// RUN: mlir-opt %s -split-input-file -tosa-validate | FileCheck %s
+
+// --- First case: without TOSA Operation (#173370 bug reproduction) ---
+
+// CHECK-LABEL: func.func @test_validation_pass_init
+func.func @test_validation_pass_init(%arg0: tensor<1xf32>) -> tensor<1xf32> {
+  // CHECK: math.asin
+  %0 = math.asin %arg0 : tensor<1xf32>
+  return %0 : tensor<1xf32>
+}
+
+// -----
+
+// --- Second case: with TOSA Operation ---
```

Comment:
What is this test-case doing on top of existing tests like in `mlir/test/Dialect/Tosa/tosa-validation-valid.mlir`?